s3watcher/deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
This file has been replaced with [`build.sh`](https://github.com/guardian/grid/blob/master/s3watcher/build.sh), might need to rebase master?